sysutils/eligibility-util/Portfile

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           xcode 1.0
+
+github.setup        Kyle-Ye eligibility 0.2.1-beta
+name                eligibility-util
+revision            0
+categories          sysutils
+maintainers         {i0ntempest @i0ntempest} openmaintainer
+platforms           {macosx >= 24}
+license             MIT
+
+description         Open source macOS system eligibility tweaking utility
+long_description    ${description}. Can be used for enabling features unavailable in certain Mac SKUs, \
+                    e.g. region locked features like Apple Intelligence.
+
+checksums           rmd160  fcd6158935b6b0e84a06e57800fe6a6e9b4fbca8 \
+                    sha256  56bd6be6de38654a78bccd370d2f2cfa9fb2b3dd950f794c2f1b326101ba9b34 \
+                    size    380169
+
+if {${os.major} < 24} {
```

Comment:
as usual, this should also check for os.platform

Reply:
Doesn’t the platforms line at the top already restricts installation to macOS only?

Reply:
You're probably right, I missed that one (and assumed it was the default `darwin`)